src/style.css

```diff
@@ -0,0 +1,53 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap&subset=cyrillic");
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.logo {
+  margin: 10px auto 10px 50px;
+}
+
+.navbar {
+  margin-right: 50px;
+  display: flex;
+}
+
+.nav_link {
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
+  color: #000;
+  margin-left: 20px;
+}
+
+.nav_link:hover,
+.is-active {
+  color: #00acdc;
+}
+
+.is-active {
+  margin-left: 0;
```

Comment:
if we add this class to another element of the list?

Reply:
![image](https://user-images.githubusercontent.com/44294945/64179174-656d5f80-ce6b-11e9-8129-a91ba983f5b5.png)